Author: Damyan Ivanov <dmn@debian.org>
Description: Can't locate object method "send_http_header"
 Trivial patch to remove a stale reference to send_http_header that is no
 longer available or necessary
Last-Update: 2011-03-22
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=507821
Forwarded: yes
Reviewed-by: Nicholas Bamber <nicholas@periapt.co.uk>
--- a/docs/api/ModPerl/Registry.pod
+++ b/docs/api/ModPerl/Registry.pod
@@ -31,7 +31,6 @@
 
   my $r = Apache2::RequestUtil->request;
   $r->content_type("text/html");
-  $r->send_http_header;
   $r->print("mod_perl rules!");
 
 XXX: STOPPED here. Below is the old Apache::Registry document which I
